comp/prizes/acquire-prizes-edible.yaml

```
@@ -0,0 +1,24 @@
+summary: Organise edible prizes for the competition
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+area-owner: production
+
+description: >-
+  The prizes for the competition are a combination of trophies and edible prizes.
+  This task only considers the edible prizes; there is a separate task for trophies.
+
+
+  We usually have around two awards that have edible prizes.
+  Typically we give a box of celebrations and a box of jelly babies as the edible prize.
```

Suggestion: maybe note something about the motivations here, rather than just what we tend to do? Including the motivation enables the person doing the task to be better informed in case we want to change things.

```suggestion
  We usually have around two awards that have edible prizes.
  We should ensure that these account for a variety of tastes and dietary requirements.
  Typically we give a box of celebrations and a box of jelly babies as the edible prize.
```